### PR TITLE
developer setting for unlimited map zooming

### PR DESCRIPTION
**Pull request by garvankeeley**

### CODE REVIEW

File: android/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -195,4 +195,5 @@
 \n<li>New testing infrastructure for developers</li>
 \n<li>Simulation mode added to the developer options.</li>
 </string>
+    <string name="limit_map_zoom">Limit map zoom</string>
```
Comment (crankycoder) [inline android/src/main/res/values/strings.xml]: Nit: "Limit zoom out" would be clearer.

**Comment by crankycoder** [comment]
Other than the wording on the string, this works for me and is fine.  r+

File: android/src/main/res/values/strings.xml (hunk 1 above)
Comment (garvankeeley) [inline android/src/main/res/values/strings.xml]: "Limit zooming out on map" ?

Comment (garvankeeley) [inline android/src/main/res/values/strings.xml]: Let's try that and see how it goes. I think this will get moved to Settings if it is liked.

**Pull request merged**